test/testutil.js

```diff
@@ -306,6 +308,48 @@ async function assertParamsWithMultipleRenditions(receivedMetrics) {
     },{
         eventType: "activation"
     }]);
+
+    assertDurationMetrics(receivedMetrics);
+}
+
+function assertDurationMetrics(receivedMetrics) {
+    receivedMetrics.filter(metric => metric.eventType === "rendition").forEach(metric => {
+        // must be positive values
+        assert.ok(metric.downloadDuration > 0.0);
```

Comment:
This is float comparision - How small are the values? Does `metric.downloadDuration > 0.0` returns true for `metric.downloadDuration === 0.001`?
Also, I maybe would rather write `0.000` in the test to indicate we expect 3 digits.

Reply:
Yes it works for small values - they are very small in our unit tests and it works.

> we expect 3 digits

The 3 digit precision only applies when logged as string. Here it compares the value, and 0.0 is the same as 0.000.

Reply:
My suggestion was not technical, but if I don't look at documentation, I look at unit tests. Having 0.000 instead 0.0 would hint me that we have 3 digits précision.

(It doesn't change the code - logic, as you said. It's just a hint for people reading tests when looking for examples).

Reply:
But the most important use of these measurements is in our metrics where we have full floating point precision (whatever New Relic supports).